docs/framework-snyk.md

```diff
@@ -0,0 +1,50 @@
+# Snyk Framework
+The Snyk Framework causes an application to be automatically configured to work with a bound [Snyk Service][].
+Binding an application to the service will cause the buildpack to check for vulnerable dependencies and break the build process
+if found any, for a given severity threshold.
+
+<table>
+  <tr>
+    <td><strong>Detection Criterion</strong></td><td>Existence of a bound Snyk service.
+      <ul>
+        <li>Existence of a Snyk service is defined as the <a href="http://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-SERVICES"><code>VCAP_SERVICES</code></a> payload containing a service who's name, label or tag has <code>snyk</code> as a substring.</li>
+      </ul>
+      <ul>
+        <li>Existence of an `apiToken` value Configuration.
```

Comment:
Markdown doesn't work inside of the `<table>` declaration.  This should be `<code></code>` instead.